base/float.jl

```diff
@@ -750,17 +750,30 @@ end
 end
 
 """
-    floatmin(T)
+    floatmin([T])
```

Comment:
```suggestion
    floatmin(T = Float64)
```
Then it does not need to be mentioned separately below.